spec.html

```diff
@@ -30185,6 +30198,8 @@ <h1>
               1. Let _bodySym_ be the grammar symbol |AsyncGeneratorBody|.
               1. Let _parameterSym_ be the grammar symbol |FormalParameters[+Yield, +Await]|.
               1. Let _fallbackProto_ be *"%AsyncGeneratorFunction.prototype%"*.
+            1. Let _currentRealm_ be the current Realm Record.
+            1. Perform ? HostEnsureCanCompileStrings(_currentRealm_, _bodySym_, _bodyArg_).
```

Comment:
`_bodyArg_` is an ECMAScript language value, but HostEnsureCanCompileStrings says it expects a String.